Files/Scripts/Options/Majora's Mask.psm1

```diff
@@ -1256,13 +1256,24 @@ function ByteTextOptions() {
-        SetMessage -ID "004C" -Text "<W> Buttons" -Replace "<W>"; SetMessage -ID "0059" -Text ", then press that <N><C><C Button> <W>Button to use it." -Replace "<W>."; SetMessage -ID "1700" -Text "<C Button><N>Buttons. Press" -Replace "<C Button>.<N>Press" # Buttons -> C Icon Only	    
-        
-        SetMessage -ID "001A" -Text "Press" -Replace "Use"; SetMessage -ID "0022" -All; SetMessage -ID "0028"; SetMessage -ID "0029"; SetMessage -ID "002A"; SetMessage -ID "002E"; SetMessage -ID "0034"; SetMessage -ID "0036"; SetMessage -ID "003A"; SetMessage -ID "0041"; SetMessage -ID "0043"; SetMessage -ID "1701" # Press -> Use
-        SetMessage -ID "1706";                              SetMessage -ID "1709";      SetMessage -ID "170E"; SetMessage -ID "170F"; SetMessage -ID "1717"; SetMessage -ID "171B"; SetMessage -ID "172E"; SetMessage -ID "0079"; SetMessage -ID "007A"
+	SetMessage -ID "004C" -Text "four <C><C Button><W> Buttons to play it. Press<N><B Button> to stop." -Replace "four <C><C Button><W> directions to play it.<N>Press <B Button> to stop."
+        SetMessage -ID "1700" -Text "<C Button><N>Buttons. Press" -Replace "<C Button>.<N>Press"
+	
+        SetMessage -ID "001A" -Text "Press" -Replace "Use";      SetMessage -ID "0022" -All; SetMessage -ID "0028"; SetMessage -ID "0029"; SetMessage -ID "002A"; SetMessage -ID "002E"; SetMessage -ID "0034"; SetMessage -ID "0036"
+        SetMessage -ID "003A"; SetMessage -ID "0041"; SetMessage -ID "0078"; SetMessage -ID "1701"; SetMessage -ID "1706"; SetMessage -ID "1709"; SetMessage -ID "170E"; SetMessage -ID "170F"; SetMessage -ID "1717"
+	SetMessage -ID "171B"; SetMessage -ID "172E"
+
+        SetMessage -ID "0014" -Text "press" -Replace "use"; SetMessage -ID "0015"; SetMessage -ID "0016"; SetMessage -ID "0017"; SetMessage -ID "0018"; SetMessage -ID "0019"; SetMessage -ID "0035"; SetMessage -ID "0200"; SetMessage -ID "09E3"
+	SetMessage -ID "1F4F"
+
+        SetMessage -ID "0078" -Text "the body of a Deku. Press <C><C Button>" -Replace "the body of a Deku. Use <C><C Button>"
+	SetMessage -ID "0079" -Text "the body of a Goron. Press <C><C Button>" -Replace "the body of a Goron. Use <C><C Button>"
+        SetMessage -ID "007A" -Text "the body of a Zora. Press <C><C Button>" -Replace "the body of a Zora. Use <C><C Button>"
+
+        SetMessage -ID "0043" -Text "Press <C><C Button> <W>to look through it" -Replace "Use <C><C Button> <W>to look through it"
+	SetMessage -ID "0059" -Text "then press that <N><C><C Button> <W>Button to use it." -Replace "then use <N><C><C Button><W>."
+
+	if ( (IsIndex -Elem $Redux.Features.OcarinaIcons.selectedIndex -Text "Enabled") -or (IsIndex -Elem $Redux.Features.OcarinaIcons.selectedIndex -Text "Enabled with Original Icon") )   { SetMessage -ID "1726" -Text "<C Button><N>Buttons. Press" -Replace "<C Button>.<N>Press"; SetMessage -ID "171C"; SetMessage -ID "1727" }
```

Comment:
I suggest integrating this check with the actual Ocarina Icons option (line 1444, not accounting for your added lines). And either run the the previous text boxes or the new adjusted ones by you if the GC Scheme option is enabled.

That way this box is only ran once and not twice.

`IsDefault $Redux.Features.OcarinaIcons -Not` might also be a shorter command to run instead of two separate checks.

Reply:
> `IsDefault $Redux.Features.OcarinaIcons -Not` might also be a shorter command to run instead of two separate checks.
> 
> Also, this check comes too early in the code. The text boxes were unused and only set to use Ocarina messages at line 1444 (not accounting for your added lines). Meaning the text boxes don't have the adjusted text just yet you want to further adjust when it reaches your check.
> 
> I would also prefer it if it could be avoided to adjust the same textbox more than once. Try if you can insert this check from line 1444 instead.
> 
> Please confirm if your changes work ingame using your branch.
> 
> Other than the Ocarina Icons text boxes I see no further issues to deny approval. Looks fine to me otherwise.

I did try the line you spoke about at the beginning and it gave me errors.
Yeah, it may looks strange, but I tested mine and it works. I did put it at line **_1466_** instead though, as it was too early indeed.

Just as thing to remember, the patcher is patching texts as the last step before **ROM** compression, so I used this as my advantage, since the **Ocarina Icons** option get patched before it and it consider it a script, rather than simple text replacement.

Reply:
> > snips

Also, if you can do something more, remove the **ACSII** lines at:

_IsChecked $Redux.Capacity.EnableAmmo_ and _IsChecked $Redux.Capacity.EnableWallet_, as they are just text replacements, not heavier scripts.

Reply:
I believe I included the ASCII parameter since it's only digit text replacing. ASCII should prevent any texts from being parsed, sparing on processing time. If I recall correctly.

Reply:
> I believe I included the ASCII parameter since it's only digit text replacing. ASCII should prevent any texts from being parsed, sparing on processing time. If I recall correctly.

Excepted, text in **ACSII** reads differently. For example, the number "**10**" reads "**49 48**" in **ACSII**. So, if anything, you could convert the simple quantity text to : 

if (IsChecked $Redux.Capacity.EnableAmmo -Lang 1) {
        SetMessage -ID "0019" -ASCII -Text "49 48" -Replace $Redux.Capacity.DekuSticks1.text
        SetMessage -ID "178A" -ASCII -Text "51 48 32" -Replace $Redux.Capacity.Quiver1.text
        SetMessage -ID "178B" -ASCII -Text "52 48" -Replace $Redux.Capacity.Quiver2.text;  SetMessage -ID "0023"
        SetMessage -ID "178C" -ASCII -Text "53 48" -Replace $Redux.Capacity.Quiver3.text;  SetMessage -ID "0024"
        SetMessage -ID "178D" -ASCII -Text "50 48" -Replace $Redux.Capacity.BombBag1.text
        SetMessage -ID "178E" -ASCII -Text "51 48" -Replace $Redux.Capacity.BombBag2.text; SetMessage -ID "001C"
        SetMessage -ID "178F" -ASCII -Text "52 48" -Replace $Redux.Capacity.BombBag3.text; SetMessage -ID "001D"
    }
    
if (IsChecked $Redux.Capacity.EnableWallet -Lang 1) {
        SetMessage -ID "0008" -ASCII -Text "50 48 48" -Replace $Redux.Capacity.Wallet2.text
        SetMessage -ID "0009" -ASCII -Text "53 48 48" -Replace $Redux.Capacity.Wallet3.text
    }

So, it will read it accordingly.

Reply:
Oh. It is always parsed into ASCII. But I was referring to ASCII parser as defined by OoT/MM itself. Both games use a different format of ASCII than regular ASCII. Such as special commands or regional letters. Setting the ASCII parameter prevents this special OoT/MM ASCII parser, not the regular ASCII parser.

Reply:
> snips

Well, what we gonna do? Because, as it stands, when changing the numbers in these quantities options, the console indicates it cannot find them.

That's why I complained in the first place.

Reply:
Lol. I wasn't aware it was broken. I fix the Text Editor then.